dali/test/python/test_operator_brightness_contrast.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nvidia.dali as dali
+import nvidia.dali.fn as fn
+from nvidia.dali import pipeline_def
+import nvidia.dali.types as types
+import numpy as np
+from test_utils import compare_pipelines
+from test_utils import RandomDataIterator
+
+def max_range(dtype):
+  if dtype == np.half or dtype == np.single or dtype == np.double:
+    return 1.0
+  else:
+    return np.iinfo(dtype).max
+
+def type_range(dtype):
+  if dtype in [np.half, np.single, np.double]:
+    return (np.finfo(dtype).min, np.finfo(dtype).max)
+  else:
+    return (np.iinfo(dtype).min, np.iinfo(dtype).max)
+
+def convert_sat(data, out_dtype):
+  clipped = np.clip(data, *type_range(out_dtype))
+  if out_dtype not in [np.half, np.single, np.double]:
+    clipped = np.round(clipped)
+  return clipped.astype(out_dtype)
+
+def dali_type_to_np(dtype):
+  if dtype == types.FLOAT:
+    return np.single
+  elif dtype == types.INT16:
+    return np.short
+  elif dtype == types.INT32:
+    return np.intc
+  elif dtype == types.UINT8:
+    return np.ubyte
+  else:
+    assert False
+
+def bricon_ref(input, brightness, brightness_shift, contrast, contrast_center, out_dtype):
+  output_range = max_range(out_dtype)
+  input_range = max_range(input.dtype)
+  norm = output_range / input_range
+  output = brightness_shift * output_range + norm * brightness * (contrast_center + contrast * (input - contrast_center))
+  return convert_sat(output, out_dtype)
+
+def ref_operator(contrast_center, out_dtype):
+  return lambda input, brightness, brightness_shift, contrast: \
+                bricon_ref(input, brightness, brightness_shift, contrast, contrast_center, out_dtype)
+
+def contrast_param():
+  return fn.random.uniform(range=[-1.0, 1.0], seed=123)
+
+def brightness_params():
+  return fn.random.uniform(range=[0.0, 5.0], seed=123), \
+         fn.random.uniform(range=[-1.0, 1.0], seed=123)
+
+@pipeline_def(num_threads=4, device_id=0, seed=1234)
+def bri_pipe(data_iterator, dtype, dev='cpu'):
+  brightness, brightness_shift = brightness_params()
+  inp = fn.external_source(source=data_iterator)
+  if dev == 'gpu':
+    inp = inp.gpu()
+  inp = fn.brightness(inp, brightness=brightness, brightness_shift=brightness_shift, dtype=dtype)
+  return inp
```

Comment:
```suggestion
  out = fn.brightness(inp, brightness=brightness, brightness_shift=brightness_shift, dtype=dtype)
  return out
```
And similar in other functions.
Just a suggestion.

Reply:
Changed to directly returning